### DISCUSSION TITLE
Nostr plugin does not display pubkey and relays

### DISCUSSION THREAD

**Discussion by Sjors** [btcpayserver/btcpayserver#5978] [Plugins & integrations]
I just upgraded the plugin to 1.1.4.0 (forgot which version it was at).

I entered my nsec and a bunch of relays before. None of them show up:

<img width="1158" alt="Scherm­afbeelding 2024-05-10 om 19 45 07" src="https://github.com/btcpayserver/btcpayserver/assets/10217/bc8c54e1-9902-4ec5-ba7c-cecf137992e9">

I added them again, and they briefly showed up and then disappeared.

But if I look at the logs it does seem zaps to me are processed:

```
May 10 17:42:08 sjimmie dotnet[378758]: info: PayServer:      BTC (Lightning): Payment detected via notification (F3axoC...)
May 10 17:42:09 sjimmie dotnet[378758]: info: BTCPayServer.Plugins.NIP05.Zapper: Attempting to send 1 zap receipts
May 10 17:42:10 sjimmie dotnet[378758]: info: BTCPayServer.Plugins.NIP05.Zapper: Zapping 1 to wss://eden.nostr.land
May 10 17:42:10 sjimmie dotnet[378758]: info: BTCPayServer.Plugins.NIP05.Zapper: Zapping 1 to wss://relay.damus.io
````

However they don't seem to arrive, at least I don't see them in Damus or if I look at my profile on the Primal website. Receiving zaps did work before I upgraded.

Nostr plugin 1.1.4.0
BTCPay Server v1.13.1+ca55e1f300a83ede660b945e9ca48c7262133f1e (Docker free install)

**Comment by remcoros** [comment]
Did you figure this out yet? I'm seeing the same, I can receive zaps, according to the logs they are being sent to relays, but I don't see them in any nostr client.

**Reply by mp21-org** [reply]
that issue I have as well. I never know who zapped me.
